aredis/commands/streams.py

```diff
@@ -29,8 +29,10 @@ def list_of_pairs_to_dict(response):
 
 def parse_xinfo_stream(response):
     res = pairs_to_dict(response)
-    res['first-entry'][1] = pairs_to_dict(res['first-entry'][1])
-    res['last-entry'][1] = pairs_to_dict(res['last-entry'][1])
+    if res['first-entry'] and len(res['first-entry']) > 0:
```

Comment:
Actually i think it could be checked by simply `if res['first-entry']:`, judgement of the length is not needed here. How do you think about it?

Reply:
`first-entry`and `last-entry` seem to be null when the stream is empty, and be of length 2 when there is data present, so I guess it's enough to check for null.